src/types/Patterns.ts

```diff
@@ -23,8 +23,55 @@ export const Patterns = {
   CITY: /^[A-ZÀ-ÿ0-9Śął](([ .'-]|\. )?[A-Za-zÀ-ÿ0-9Śął]{1,40}){1,10}$/,
   STREET:
```

Comment:
Can we delete the "Street" pattern? It seems to be not used at all

Reply:
@jjeroch it's used for a mandatory field in the company data page

Reply:
@oyo @jjeroch Seems like streetHouseNumberPattern is not used.

Reply:
@typecastcloud yes - please remove